tests/test_orm.py

```diff
@@ -71,6 +71,49 @@ def test_model_basics():
     assert record["fields"]["First Name"] == contact.first_name
 
 
+def test_model_save_specific_fields():
```

Comment:
@BAPCon Could be nice to add an additional test that fails the `Model.save` and `Model.batch_save` when an undeclared Model ORM field is passed in in the `fields`.